boards/nrf51dk/README.md

```diff
@@ -37,28 +61,35 @@ using `JLinkExe`.
 
 ## Debugging
 
-To manually connect to the board with a jlink programmer:
+Because the nRF51DK has integrated JTAG support, you can
+program it directly using gdb. In this setup, gdb connects
+to a process that gives access to the device over JTAG.
+First, create a `.gdbinit` file in the directory you are
```

Comment:
I pulled the text for the gdbinit from the end of the instructions; given you needed to have it at the beginning (e.g., when you run gdb), it made more sense to put it earlier.